urnai/models/model_base.py

```diff
@@ -10,7 +10,7 @@ def __init__(self):
 		self.learning_data = {}
 	
 	@abstractmethod
-	def learn(self, current_state, action, reward, next_state):
+	def learn(self, current_state, action, reward, next_state, done):
```

Comment:
When testing the `learn` method from `agent_base` class with a `previous_state` set, the method calls the `learn` method from `model_base`, and passed an extra parameter `done` to it. So, I had to add this parameter to the original abstract method from `model_base` class, and had to add it to the method that tests their methods too. (`test_model_base`)

Another way to solve this would be removing `done` from the call made inside agent_base's `learn` method, but I think that this paramater should be important in the future when implementing the subclasses for it.